conf/config.properties

```diff
@@ -21,20 +21,19 @@ users.password.regex.tooltip=Enter a combination of atleast six characters
 users.admin.localonly=false
 
 # SMTP emailing switch (for password recovery and signup verification etc)
-smtp.mails.enabled=true
-
-# SMTP host
-smtp.host.name=smtp.gmail.com
+mails.type=SMTP
 
 # SMTP connection encryption and corresponding port (none/25, starttls/587 or tls/465). in unusual cases, different ports may be used.
 smtp.host.encryption=tls
 smtp.host.port=465
 
 # SMTP sender displayname (the name displayed to the recipient), email, username (to login) and password. Usually email and username are the same (but not necessarily)
-smtp.sender.displayname=Admin
-smtp.sender.email=server@loklak.org
+mail.displayname=Admin
+mail.email=smtp.gmail.com
```

Comment:
where does this come from? This is not an email! Do you mean to create smtp gateways? Please check how eventyay does it.

Reply:
Okay, will look into it